src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/__tests__/Survey/SurveyConsentViewModelTests.swift

```diff
@@ -8,14 +8,17 @@ import XCTest
 class SurveyConsentViewModelTests: XCTestCase {
 
 	func testDynamicTableViewModel() {
-		let placeHolderString = "https://www.test.de"
-		let viewModel = SurveyConsentViewModel(urlString: placeHolderString)
+		if let url = URL(string: "https://www.test.de") {
```

Comment:
No need to nag the Stiftung Warentest servers!
```suggestion
		if let url = URL(string: "https://www.example.com") {
```